Owncloud iOs Client/DataBase/DTOs/CredentialsDto.h

```diff
@@ -17,8 +17,12 @@
 
 @interface CredentialsDto : NSObject
 
+@property (nonatomic, copy) NSString *userId;
 @property (nonatomic, copy) NSString *userName;
-@property (nonatomic, copy) NSString *password;
+@property (nonatomic, copy) NSString *password; //or accessToken in oauth
```

Comment:
I would call it ```accessToken```, seems more general; then explain in the comment that password goes there for basic auth.